### DISCUSSION TITLE
Is it a way to show duration for the firing alerts?

### DISCUSSION THREAD

**Discussion by egorvas** [metalmatze/alertmanager-bot#148] [Help/Support]
Hi.
As I see, we can show duration only for the resolved alerts(regarding templates and duration function) between startsAt and endsAt.
But I would like to show a duration between startsAt and time.Now for the currently firing alerts.
Do we have a solution for it? The only way that I found it is fork and fix duration function(set default value for end parameter), but its too complicated for my task.

**Comment by metalmatze** [comment]
That might be something we can totally add. I didn't really think about that. It sounds like a nice enhancement though.
Feel free to create a PR with your changes! Happy to have them.
